tests/fixture/tmpnet/network.go

```diff
@@ -1132,3 +1142,59 @@ func MetricsLinkForNetwork(networkUUID string, startTime string, endTime string)
 		endTime,
 	)
 }
+
+// applyRandomizedFlags applies randomized network parameters using the provided seed
+func (n *Network) applyRandomizedFlags(log logging.Logger, antithesisSeed string) {
+	// Validate the seed format (the actual randomization is done in GetRandomizedParams)
+	if _, err := strconv.ParseInt(antithesisSeed, 10, 64); err != nil {
+		log.Warn("failed to parse ANTITHESIS_RANDOM_SEED, skipping randomization", zap.Error(err))
+		return
+	}
+
+	// TODO(jonathanoppenheimer): is there a better way to apply these values?
```

Comment:
This seemed like a bad way to apply the `randomizedParams` Is there a more concise way of doing so?